Sources/Queues/AsyncJobEventDelegate.swift

```diff
@@ -0,0 +1,60 @@
+#if compiler(>=5.5) && canImport(_Concurrency)
```

Comment:
With 5.6 as a minimum requirement, we don't need the #if's anymore.

Reply:
Everywhere? Also `_canImport(_Concurrency)`? Does that mean Vapor also technically could remove all of these `#if` since it has a minimum requirement of Swift 5.5.2?

Reply:
That is my understanding, yes, though I admit I have been meaning to ask @Lukasa about it to be 100% sure.